bridge/test/bToken/mint.test.ts

```diff
@@ -180,10 +181,12 @@ describe("Testing BToken Minting methods", async () => {
   it("Should fail to mint with big min BToken quantity", async () => {
     const oneBToken = ethers.utils.parseUnits("1", 18).toBigInt();
     const minBTokens = 900n * oneBToken;
-    await expect(
+
+    await assertErrorMessage(
       fixture.bToken.connect(admin).mint(minBTokens, {
         value: ethers.utils.parseEther(eth.toString()),
-      })
-    ).to.be.revertedWith("308");
+      }),
+      `MinimumMintNotMet(${"399028731704364116575"}, ${minBTokens})`
```

Comment:
Where does this magic number come from?

Reply:
Good point - I've refactored that as a variable with a decent name